docs/sphinx/quickstart.rst

```diff
@@ -173,7 +173,7 @@ build an executable using |cmake|_ and |hpx|:
 .. note::
 
    ``COMPONENT_DEPENDENCIES iostreams`` is optional for a minimal project but
-   lets us use the |hpx| equivalent of ``std::cout``, i.e. the |hpx|
+   lets users use the |hpx| equivalent of ``std::cout``, i.e., the |hpx|
```

Comment:
maybe `lets you` instead? Throughout the rest of text, we address the reader with "you"